tests/test_cli.py

```diff
@@ -86,7 +86,6 @@ def test_install_with_helloapache(self):
 
         json_data=open(tests_root + "cached_nulecules/helloapache/answers.conf.sample").read()
 
-        assert exec_info.value.code == 0
```

Comment:
this is not ok, you modify the test so that it passes, removing line 89 will result in a test case that is testing the json module and not the atomicapp.cli